packages/@aws-cdk/aws-lambda-nodejs/lib/bundling.ts

```diff
@@ -219,6 +219,7 @@ export class Bundling implements cdk.BundlingOptions {
 
       // Create dummy package.json, copy lock file if any and then install
       depsCommand = chain([
+        this.packageManager.lockFile === LockFile.PNPM ? osCommand.write(pathJoin(options.outputDir, 'pnpm-workspace.yaml'), ''): '',
```

Comment:
If the only instance where you're using `write` has no value for `data`, I'm not sure I get why that function takes in two values and why it's being used here. Are you just creating an empty file titled `pnpm-workspace.yaml`? Also, why use an empty string instead of undefined for this.packageManager.lockFile if it's not pnpm?

Reply:
Yes creating an empty `pnpm-workspace.yaml` file avoids a problem when using workspaces. I am now calling the `write()` function twice, once with data. I have updated the PR description to explain.

I am using an empty string as `chain()` takes an array of strings.